importer/src/main/java/org/hiero/mirror/importer/db/DBProperties.java

```diff
@@ -51,6 +51,9 @@ public class DBProperties {
     @NotBlank
     private String username = "";
 
+    @NotBlank
+    private String sslMode = "";
```

Comment:
Need to set the default. We can use the jdbc enum to provide value validation as well.

```suggestion
    @NotNull
    private SslMode sslMode = SslMode.DISABLE;
```

Reply:
Thanks! Will take care of it